.github/CHANGELOG.md

```diff
@@ -2,10 +2,19 @@
 
 ### New features since last release
 
+* 1,2-qubit `qml.QubitUnitary` operators can be included in a circuit differentiated with the adjoint method.
+  [(#540)] (https://github.com/PennyLaneAI/pennylane-lightning/pull/540)
```

Comment:
Thanks @vincentmr! What constraint adds the specificity to 1 or 2 qubit unitaries?

Reply:
At the moment we rely on ZYZ decomposition (to get a circuit with single-parameter gates) which is only implemented for 1 and 2 qubit unitaries.

Reply:
So @vincentmr does this PR use decomposition to achieve the objective? My understanding was that we'd support arbitrary number qubit unitaries just with the available unitary matrix?

Reply:
The story stated
- Validate adjoint diff works with 1,2, 3+ qubit unitaries in forward and backwards pass without trainable data
- Add support for 1 and 2 qubit unitary decomposition to allow direct support of QU with adjoint in Lightning devices
This PR fulfills these requirements. Ah I realize the changelog is outdated with respect to the unit tests. I'll update it.

Reply:
Thanks, the updated changelog is awesome!